addon/mixins/service-mixin.js

```diff
@@ -170,5 +170,26 @@ export default Mixin.create({
     .then((resp) => {
       return this.checkStatusAndParseJson(resp);
     });
+  },
+
+  /**
+   * Wrap the options passed to rest-js with auth info and use ember-fetch.
+   */
+  addOpts (args, portalOpts) {
```

Comment:
`addOpts` is perfect, and should be easy to test I think a couple of tests on this could suffice instead of stubbing and spying on rest-js fns.

Reply:
actually, it would be good to move this to it's own util (`ember g util add-options --classic`) b/c:
1. it's stateless
1. util fns > mixins (mixins--)
1. i'm interested to see if we end up using this mixin at all once everything is converted

That will scaffold a test for you too.

Reply:
sounds good to me.

there's probably a more ember-y way to maintain a reference to the active `torii` session, but for now i'm just passing through `this` as _another_ argument.

as for the tests. i _think_ i'm fairly close to scaffolding out a mock `torii` session, but the errors i've been encountering are pretty cryptic, so i decided quit typing blindly and just ask for help.